packages/experimental/Link/src/LinkTokens.android.ts

```diff
@@ -0,0 +1,26 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+import { LinkTokens } from './Link.types';
+
+export const defaultLinkTokens: TokenSettings<LinkTokens, Theme> = (t: Theme) =>
+  ({
```

Comment:
What was changed in this file?

Reply:
@ValentinaKozlova , This is entirely newly added file for Android Specific Tokens for Link.